src/rules/selector-max-specificity/README.md

```diff
@@ -0,0 +1,29 @@
+# selector-max-specificity
+
+Control the specificity of selectors.
+
+First define the maximum allowable selector specificity as computed by the [W3C specificity calculator](https://www.w3.org/TR/selectors/#specificity):
+
+For example:
+
+| inline | ID | Class | type
+|---------|----|---------|-------
+| 0 | 0 | 2 | 0
+
+Set this as a 4 digit value in your Stylelint configuration:
```

Comment:
Follow the standard ["Options"](https://github.com/stylelint/stylelint/tree/master/src/rules/max-line-length#options) layout here i.e. show patterns that are considered warnings and then another set of patterns that are not.